app/views/users/show.html.erb

```diff
@@ -1,16 +1,26 @@
+<div class= 'post-info'>
 <h1>User info</h1>
-<h3>Name: <%= @user.name %></h3>
-<h3>Bio: <%= @user.bio %></h3>
+<div class= 'users-info'>
+<img src="<%= @user.photo %>" alt="user photo" class="img">
+<div class= user-list>
+<h2><%= @user.name %></h2>
+<p class= 'postscounter'>Number of Posts: <%= @user.postsCounter %></p>
+</div>
+</div>
+<p>Bio:</p>
+<p class= 'bio'> <%= @user.bio %></p>
 
-<h1>Posts</h1>
 
-<ul>
+<h1><%= link_to 'Posts', user_posts_path(@user) %></h1>
+
+<div>
     <% @user.posts.each do |post| %>
-        <li>
-    <%= link_to post.title, user_post_path(@user, post) %>
-        </li>
+        <h3 class='user-posts'> Post #<%= post.id %> <%= link_to post.title, user_post_path(@user, post) %>
+        </h3>
+        <p><%= post.text %> </p>
+        <p class= 'user-comments-likes'>Comments: <%= post.commentsCounter %>, Likes: <%= post.likesCounter %></p>
```

Comment:
- [ ] instead of  `posts.each` you should use the method `recent_posts` that you created in the previous task also please make sure to add a user image and bio as the wireframe  to complete the requirements 
- yours 
<img width="508" alt="Screenshot 2023-05-11 at 19 47 04" src="https://github.com/The-Normal-One1/Blog-App/assets/61790587/71b8c32d-c68d-4f39-bd81-685e576b408c">
- wireframe
<img width="435" alt="Screenshot 2023-05-11 at 19 46 52" src="https://github.com/The-Normal-One1/Blog-App/assets/61790587/f818a498-d28a-4fea-a1db-2e25eed3b351">